client/flutter/lib/api/content/content_bundle.dart

```diff
@@ -62,6 +62,22 @@ class ContentBundle {
     }
   }
 
+  YamlList get contentResults {
+    try {
+      return yaml['contents']['results'];
+    } catch (err) {
+      return YamlList();
+    }
+  }
+
+  YamlList get contentCards {
+    try {
+      return yaml['contents']['cards'];
```

Comment:
If changing the name as proposed in another comment:

```suggestion
      return yaml['contents']['results_cards'];
```